WinRT.Runtime/Interop/IAgileReference.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace WinRT.Interop
+{
+    [WindowsRuntimeType]
+    [Guid("C03F6A43-65A4-9818-987E-E0B810D2A6F2")]
+    internal interface IAgileReference
+    {
+        IObjectReference Resolve(Guid riid);
+    }
+
+    [WindowsRuntimeType]
+    [Guid("94ea2b94-e9cc-49e0-c0ff-ee64ca8f5b90")]
+    public interface IAgileObject

```

Comment:
nit: fold https://github.com/microsoft/CsWinRT/blob/1dc85f4f14deb3d85f83c3fc60b6684d25ed733b/WinRT.Runtime/ComWrappersSupport.cs#L31 in with this definition